### ISSUE TITLE
Docs: add (obvious) link to Austin repo and one-line description of what it's for

### ISSUE THREAD

**Issue by clbarnes** [P403n1x87/austin-tui#9]
e.g. first sentence of the synopsis could read

```markdown
This Python TUI is a top-like text-based user interface for [Austin](https://github.com/P403n1x87/austin) (a Python frame stack sampler for CPython), written in pure Python.
```

**Comment by P403n1x87** [comment]
Thanks for your feedback! The PRs #7 and #8 address this issue.

**Comment by P403n1x87** [comment]
Fixed by #7.
